### ISSUE TITLE
Impersonate account should default to emulate

### ISSUE THREAD

**Issue by heyMP** [elmsln/elmsln#2211]
Feature request:
When impersonating an account, the user will search for an account, then hit `enter` which will take them to view the student profile rather than impersonating.  The `emulate` button should be the default button when hitting enter.

We could also make the `view` button look more like a secondary option, visually.

![screen shot 2017-08-31 at 9 07 01 am](https://user-images.githubusercontent.com/3428964/29925425-4d0a7f92-8e2e-11e7-8332-4fcb4c4a3bed.png)

**Comment by btopro** [comment]
agreed; didn't know it did that

**Comment by btopro** [comment]
Hmm.. I can't replicate the default behavior part. If I click in the field and then select a name and just hit enter it goes to masquerading instead of view profile. I did visually offset it a bit for the moment
